src/main/java/test/matrix/MatrixAndWord.java

```diff
@@ -0,0 +1,5 @@
+package test.matrix;
+
+public interface MatrixAndWord {
+    String getSequence_Of_Cells(String letters, String word);
```

Comment:
```suggestion
    String getCellsSequence(String letters, String word);
```
we don't use snake case for method names